documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -44,11 +44,11 @@ spec:
       # ...
     resources: <3>
       requests:
-        cpu: 200m
-        memory: 64Mi
-      limits:
         cpu: 500m
```

Comment:
This should probably be 1

Reply:
I updated this value to 1 as you suggested